index.html

```diff
@@ -122,6 +122,14 @@
         fill: red;
       }
 
+      .copy {
+        stroke: green;
```

Comment:
A good thought to have an assigned color for the copy tool, this is not usual in other systems.